src/widgets/component-new-edit/components/response-format/simple/simple-duree.jsx

```diff
@@ -58,9 +58,16 @@ class ResponseFormatDatatypeDuree extends Component {
               {Dictionary.durationformat2}
             </GenericOption>
 
+            <GenericOption
+              key="HH:CH"
+              value="HH:CH"
+            >
+              {Dictionary.durationformat3}
+            </GenericOption>
+
           </Field>   
           <div className="response-format-datatype-duree-minimum" >
-            <div className="response-format-datatype-duree-label-minimum" style= {{ display : formatini==='PTnHnM' || formatini==='PnYnM'? 'flex' : 'none' }}>   
+            <div className="response-format-datatype-duree-label-minimum" style= {{ display : formatini==='PTnHnM' || formatini==='PnYnM' || formatini==='HH:CH' ? 'flex' : 'none' }}>   
```

Comment:
We can maybe extract this test in a dedicated method ? In order to make this HTML easier to read/ What do you think ?

Reply:
I propose a solution of creating a variable `isDuration` and assigning it this test, allowing us to simplify the style to `display : isDuration ? 'flex' : 'none'`. What do you think?
I believe this way we can achieve better code readability and remain within the scope of this ticket. Ultimately, it could maybe be beneficial to have a solution to minimise the repetition of the style attribute (could be a method), perhaps having a specific ticket for code refactoring?